### PR TITLE
add no-node config

### PR DESCRIPTION
**Pull request by paulius-valiunas**
https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/no-nodejs-modules.md

Another optional config that would enrich the recommended config for browser-compatible packages

### CODE REVIEW

File: dist/configs/extension-exports-config.js [hunk 1]
```diff
@@ -6,10 +6,6 @@
 module.exports =
 {
   plugins: {
-    "@typescript-eslint": require("@typescript-eslint/eslint-plugin"),
```
Comment (aruniverse) [inline dist/configs/extension-exports-config.js:9]: Why are these removed?

Comment (paulius-valiunas) [reply dist/configs/extension-exports-config.js:9]: they're not used in this config ¯\_(ツ)_/¯

**Review by aruniverse — APPROVED** [review]
